docs/installing-northstar/troubleshooting.md

```diff
@@ -4,74 +4,72 @@ Generally try to first launch the vanilla game (i.e. not Northstar) if you encou
-- Check [this issue thread](https://github.com/R2Northstar/Northstar/issues/6) to see whether any of the solutions proposed there work for you.
-- Verify game files via Steam/Origin (depending on where you own the game)
-- Check Github issues if anyone else is experiencing this problem. Maybe current release has a bug.
+Don't panic! This warning seems to be caused by Origin and none of your files are actually corrupted. If you get this warning after launching the game try the following steps, closing the game before and launching it again after:
+
+* Make sure you got the newest version of Northstar. In particular, [v1.4.0 or higher](https://github.com/R2Northstar/Northstar/releases/) have this problem fixed.
+* Restart Origin\
+  Also check task manager that all Origin processes are gone before restarting it\
+  (even if you have the Steam version)
+* Restart your PC
+* Start Northstar with Origin already open
+* Start Northstar with Origin fully closed.
+* Start vanilla game and see if this works.\
+  If vanilla doesn't work either, check [this thread on EA forums](https://answers.ea.com/t5/Titanfall-2/Titanfall-2-Wont-Laumch-DLL-file-issue/td-p/5660909)
+* Check [this issue thread](https://github.com/R2Northstar/Northstar/issues/6) to see whether any of the solutions proposed there work for you.
+* Verify game files via Steam/Origin (depending on where you own the game)
+* Check Github issues if anyone else is experiencing this problem. Maybe current release has a bug.
 
 If you followed all these steps and even launching the vanilla version of the game doesn't work, your final option is to fully remove the game and reinstall it.\
 Then checking if vanilla works before reinstalling Northstar.
 
-## "Failed copying wsock32.dll from system32 [...] copy_file: Access is denied."
+## Failed copying wsock32.dll <a href="#wsock" id="wsock"></a>
 
 You are probably using EA Desktop app and it sets up game installations with no write permissions contrary to Origin.
 
-- **Solution 1**: Launch EA Desktop and the game as admin once so that it can copy that file.
-- **Solution 2**: If you know how, just copy the mentioned file manually, remembering you need to change its filename (just use some temp dir and rename there).
-- **Solution 3**: If you know how, just change the folder permissions in Properties->Security tab of your Titanfall2 install dir to let your user write.
+* **Solution 1**: Launch EA Desktop and the game as admin once so that it can copy that file.
+* **Solution 2**: If you know how, just copy the mentioned file manually, remembering you need to change its filename (just use some temp dir and rename there).
+* **Solution 3**: If you know how, just change the folder permissions in Properties->Security tab of your Titanfall2 install dir to let your user write.
 
-## Can't Join Servers (Issues with 10th+ gen Intel CPUs)
+## Can't Join Servers <a href="#intel" id="intel"></a>
```

Comment:
I'd keep the `(Issues with 10th+ gen Intel CPUs)`. There can be a multitude of reasons why someone isn't able to join a specific or any servers and keeping the reference to the CPU in the heading should help prevent someone with unrelated issues from trying this solution. ^^